packages/wrangler/src/__tests__/jest.setup.ts

```diff
@@ -3,6 +3,8 @@ import { confirm, prompt } from "../dialogs";
 import { fetchInternal } from "../cfetch/internal";
 import fetchMock from "jest-fetch-mock";
 
+jest.mock("../sentry");
```

Comment:
By the way, we should probably leave a comment here explaining that in general we don't want to run any sentry stuff in tests.